source/_integrations/heos.markdown

```diff
@@ -244,15 +268,37 @@ HEOS pushes data to Home Assistant via the local network when data and entity st
 
 ### Missing favorites
 
-#### Symptom: "IP_ADDRESS is not logged in to a HEOS account and will be unable to retrieve HEOS favorites..."
+#### Symptom: "The HEOS System is not logged in: Enter credentials in the integration options to access favorites and streaming services"
+
+The message above is logged during integration startup and the `source_list` attribute of the integration's media_player entities are empty. Attempting call the `media_player.play_media` action
+for `favorite` and `playlist` will fail. Other functionality of the integration is unaffected.
+
+##### Description
+
+To access features, such as favorites, playlists, and streaming services, the HEOS System must be logged in to your HEOS Account. This occurs when credentials are not entered in the configuration options and the HEOS System is in a logged out state.
+
+##### Resolution
+
+Enter the credentials to your HEOS Account in the [configuration options](/integrations/heos#configuration-options) if you want to access playlists, favorites, and streaming services; otherwise the logged warning can be ignored. If credentials are entered, the integration will ensure that the HEOS System remains logged in while the credentials remain valid.
```

Comment:
```suggestion
Enter the credentials to your HEOS Account in the [configuration options](#configuration-options) if you want to access playlists, favorites, and streaming services; otherwise, the logged warning can be ignored. If credentials are entered, the integration will ensure that the HEOS System remains logged in while the credentials remain valid.
```